File: src/SQL8r.Logic.DotNet/Templates/CS/AdoCodeSnippet.cs
using System;
using System.Collections.Generic;
using System.Data;
using System.Data.SqlClient;
using System.Xml;
using SQL8r.Logic.Interfaces;

namespace SQL8r
{
    public class AdoCodeSnippet : ISnippet
    {
        public DataSet GetMyData()
        {
            return GetMyDataAdo();
        }

        

        private DataSet GetMyDataAdo()
        {
            string selectSql = "SELECT * FROM dbo.DatabaseLog";
            string connectionString = "SERVER=.;DATABASE=AdventureWorks2008;Trusted_Connection=True;";

            SqlConnection cn = new SqlConnection(connectionString);
            SqlCommand dc = new SqlCommand(selectSql, cn);
            SqlDataAdapter da = new SqlDataAdapter(dc);
            DataSet ds = new DataSet();
            da.Fill(ds);

            return ds;
        }
    }
}

File: src/SQL8r.Logic.DotNet/Templates/VB/AdoCodeSnippet.vb
Imports System
Imports System.Data
Imports System.Data.SqlClient
Imports SQL8r.Logic.Interfaces

Namespace SQL8r

    Public Class AdoCodeSnippet
        Implements ISnippet

        Public Function GetMyData() As DataSet Implements ISnippet.GetMyData

            Dim selectSql As String = "SELECT * FROM dbo.DatabaseLog"
            Dim connectionString As String = "SERVER=.;DATABASE=AdventureWorks2008;Trusted_Connection=True;"

            Dim cn As SqlConnection = New SqlConnection(connectionString)
            Dim dc As SqlCommand = New SqlCommand(selectSql, cn)
            Dim da As SqlDataAdapter = New SqlDataAdapter(dc)
            Dim ds As DataSet = New DataSet()
            da.Fill(ds)

            Return ds
        End Function
    End Class
End Namespace